cpp/include/raft/stats/weighted_mean.cuh

```diff
@@ -159,13 +159,13 @@ void row_weighted_mean(const raft::handle_t& handle,
 
 /**
  * @brief Compute the column-wise weighted mean of the input matrix with a
- * vector of row weights
+ * vector of col weights
```

Comment:
col weights should be correct, though you might want to add it's only correct when `along_rows==true`, otherwise it's row-wise weights.